### ISSUE TITLE
"next" version of "@theia/plantuml" not found on npm?

### ISSUE THREAD

**Issue by marcdumais-work** [theia-ide/theia-plantuml-extension#7]
Having "latest" as part of a Theia application works as expected, but the "next" version is not found: 

``` js
error An unexpected error occurred: "Couldn't find any versions for \"@theia/plantuml\" that matches \"next\"".
```

**Comment by AlexTugarev** [comment]
@marcdumais-work, there is currently no CI build setup for `@theia/plantuml` which would produce a `next` version. I assume, that is the purpose for `next`, right? therefore, `next` is not published.

Just asking to understand the issue, but why do you require `next` of `@theia/plantuml`?

**Comment by marcdumais-work** [comment]
@AlexTugarev I do not know if it's on purpose or not. Maybe just waiting for a stated need?

I think it's useful to have a "next" version of this extension if we want to include plantuml support in Theia applications based on "next".  We could then add this extension in "theia-full-docker" image from the "theia-apps" repo. We have our own internal version of that Docker image that we use for demos, and it's then nice to have a more cutting-edge version of everything.

**Comment by AlexTugarev** [comment]
@marcdumais-work I hope we are on the same page here. I think we can keep this as a requirement for a CI setup, but as long as it's published manually there is no plan so far to publish upcoming releases. 

> I think it's useful to have a "next" version of this extension if we want to include plantuml support in Theia applications based on "next". We could then add this extension in "theia-full-docker" image from the "theia-apps" repo. We have our own internal version of that Docker image that we use for demos, and it's then nice to have a more cutting-edge version of everything.

Doesn't it work with `latest` of this extension? 

Also, its a bit of a chicken and egg problem here, right? If there are breaking changes in `next` of theia dependencies, they also needs to be resolved here to be republished as `next` asap, otherwise it would break the aggregating "theia-full-docker".

I'm open for any suggestions here. Maybe we should create a CI job to publish `next` automatically?

**Comment by marcdumais-work** [comment]
> Doesn't it work with latest of this extension?

It compiles, but the extension doesn't seem to work

> Also, its a bit of a chicken and egg problem here, right? If there are breaking changes in next of theia dependencies, they also needs to be resolved here to be republished as next asap, otherwise it would break the aggregating "theia-full-docker".

True, as it is for all our "external" extensions. This happened recently with the yangster extension, that's also part of the full docker image. 

> Maybe we should create a CI job to publish next automatically?

+1

**Comment by AlexTugarev** [comment]
resolved in #11
